tensorboard/plugins/pr_curve/pr_curve_op.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from collections import namedtuple
+import json
+import os
+
+import numpy as np
+import tensorflow as tf
+
+from tensorboard.plugins.pr_curve import pr_curve_plugin_data_pb2
+
```

Comment:
OK - now just one empty line between "from tensorboard.plugins.pr_curve import pr_curve_plugin_data_pb2" and the function def.